comptoir-ejb/src/main/java/be/valuya/comptoir/model/commercial/ItemVariantSale.java

```diff
@@ -48,6 +42,9 @@
     @NotNull
     @Nonnull
     private Stock stock;
+    @Column(name = "FORCE_CUSTOMER_LOYALTY")
+    @CheckForNull
+    private Boolean forceCustomerLoyalty;
```

Comment:
Finally I just renamed the field but kept it nullable :
- The variant is saved as soon as it is added to a sale
- The default value for the 'includeLoyalty' field might depend on other fields, so we need to check it on sale close.
- The user might want to overwrite it before closing the sale.